config/controlled_vocabulary.yml

```diff
@@ -1,5 +1,23 @@
 units:
   - Default Unit
+  - Aboriginal Multi-Media Society of Alberta (AMMSA)
+  - Anthropology, Department of
+  - Art/Research International
+  - Campus Data Summit Series
+  - Cultural and Pedagogical Inquiry
+  - Edmonton Mennonite Centre for Newcomers (EMCN)
+  - Graduate Studies and Research, Faculty of
+  - Grant Assist Program (Natural Sciences and Engineering)
+  - Grant Assist Program (Social Sciences / Humanities)
+  - Kule Folklore Centre
+  - Music, Department of
+  - Intonations
+  - Physics, Department of
+  - School of Library and Information Studies
+  - Secondary Education, Department of
+  - University of Alberta Archives
+  - University of Alberta Library
+  - Writing Stick
```

Comment:
@seanluyk Double checking, is this list of units in Avalon correct? Checking if any changes before asking the dev team for review.

Reply:
@jefferya this looks good to me with a recent addition needed for "Biological Sciences, Department of", but I'm flagging @anayram to make sure we're not missing anything.